src/main/kotlin/no/skatteetaten/aurora/gobo/graphql/Instrumentation.kt

```diff
@@ -80,15 +80,8 @@ class GoboInstrumentation(
                 }
             }
 
-            if (queryText.isNotIntrospectionQuery()) {
-                queryCache?.let { cache ->
-                    val request = context.request
-                    cache.add(request.klientid(), request.korrelasjonsid(), queryText)
-                }
-
-                if (logQueries == true) {
-                    logger.info { queryText }
-                }
+            if (queryText.isNotIntrospectionQuery() && logQueries == true) {
```

Comment:
Lærer stadig noe nytt: Introspection is the ability to query which resources are available in the current API schema.